lib/src/components/NxDonutChart/types.ts

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import * as PropTypes from 'prop-types';
+
+export interface Props {
```

Comment:
Should probably allow all svg attributes rather than just id in particular.  See the `types.ts` files of `NxButton` and `NxTableBody` for various ways to include native attrs in your allowed props.